src/components/left-side-navbar/LeftSideNavbar.jsx

```diff
@@ -1,7 +1,8 @@
 import { useState } from "react";
 import classNames from "classnames";
+import { useProfile } from "context";
 import { LeftSideNavbarData } from "data";
-import { useProfile, useUser } from "context";
+import { useSelector } from "react-redux";
 import { EllipsisHorizontalIcon } from "assets";
 import styles from "./LeftSideNavbar.module.css";
 import { ROUTE_HOME, ROUTE_PROFILE } from "utils";
```

Comment:
What are these variables exactly for

Reply:
These constants are for maintaining the route path.